code/modules/goals/station_goals/bluespace_tap.dm

```diff
@@ -213,6 +220,7 @@
 	new /datum/data/bluespace_tap_product("Unknown Snack", /obj/effect/spawner/lootdrop/bluespace_tap/food, 6000),
 	new /datum/data/bluespace_tap_product("Unknown Cultural Artifact", /obj/effect/spawner/lootdrop/bluespace_tap/cultural, 15000),
 	new /datum/data/bluespace_tap_product("Unknown Biological Artifact", /obj/effect/spawner/lootdrop/bluespace_tap/organic, 20000),
+	new /datum/data/bluespace_tap_product("Unknown Exotic Materials", /obj/effect/spawner/lootdrop/bluespace_tap/mats, 50000),
```

Comment:
Price likely wont be high enough still ngl. Not sure though.

Reply:
it is high enough from a normal view